travis/install-boost.sh

```diff
@@ -16,6 +16,11 @@ case "$os" in
         # We do not test Boost.Compute on Linux because of OpenCL issues...
         # Boost.Compute is a header-only library
         #git clone --depth 1 https://github.com/kylelutz/compute.git ${TRAVIS_ROOT}/compute
-        #git clone --depth 1 https://github.com/boostorg/compute.git ${TRAVIS_ROOT}/compute
+        for sp in  circular_buffer compute config core log array multi_array optional \
+                   preprocessor type_index utility assert static_assert exception throw_exception \
+                   concept_check type_traits iterator mpl detail functional move range ; do
+            git clone --depth 1 https://github.com/boostorg/${sp}.git ${TRAVIS_ROOT}/${sp}
```

Comment:
There is no reasonable option to use a package manager for this?

Reply:
I tried but it was giving me trouble, perhaps because the Boost version in the package manager wasn't new enough.  I really like using the latest versions anyways.